app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getTopics } = require("./controllers/nc-news.consrollers");
+
+const app = express();
+app.use(express.json());
```

Comment:
No need for this express.json() line yet as we aren't handling a post or patch currently.